src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    Random random = new Random();
+    int randomInt = random.nextInt(100);
+
+    public Ball getRandomBall() {
+        Ball randomBall = new Ball();
+        randomBall.setColor(randomColor);
+        randomBall.setNumber(randomInt);
```

Comment:
Move the random color and number generation inside the `getRandomBall()` method to ensure each `Ball` object has a unique color and number. Call `getRandomColor()` and `random.nextInt(100)` within the method.